### ISSUE TITLE
FatalThrowableError on first contenttype with a relation when running app/nut database:export

### ISSUE THREAD

**Issue by doenietzomoeilijk** [bolt/bolt#7811]
While migrating a site from sqlite to a mysql backend, I wanted to export the data in the sqlite database to an export file, as I've done before. However, that now results in a fatal error.

Details
-------

| Question                      | Answer
|----------------- | ------
| Relevant Bolt Version | 3.6.10, ran composer update on both Bolt and the extensions.
| Install type                   |  composer
| BC Break                     | n/a
| PHP version                 | 7.3.10 (homebrew)
| Web server                  | n/a, CLI

Reproduction
------------

If you're filing a bug, please describe how to reproduce it. Include as much
relevant information as possible, such as:

### Bug summary

Error message:

```FatalThrowableError in LazyCollection.php line 24:
Call to a member function getContenttype() on null

in LazyCollection.php line 24
at LazyCollection->serialize() in Export.php line 158
at Export->addRecord(object(MutableBag), object(ClassMetadata), object(Content)) in Export.php line 130
at Export->getRecords('themes', object(MutableBag), object(MutableBag)) in Export.php line 67
at Export->run(array('themes', 'subjects', 'publications', 'events', 'newsletters', 'wizards', 'wizard_steps', 'referrals', 'specialists', 'homepage', 'pages', 'blocks'), object(MutableBag), false) in DatabaseExport.php line 62
at DatabaseExport->execute(object(ArgvInput), object(ConsoleOutput)) in Command.php line 245
at Command->run(object(ArgvInput), object(ConsoleOutput)) in Application.php line 853
at Application->doRunCommand(object(DatabaseExport), object(ArgvInput), object(ConsoleOutput)) in Application.php line 185
at Application->doRun(object(ArgvInput), object(ConsoleOutput)) in Application.php line 117
at Application->run() in nut line 19```

From what I can tell from the dump, stuff goes pear-shaped on the first record of the first contenttype (`themes` in this case), right after the regular fields have been figured out and the script starts working on the relations.

### Steps to reproduce

Run `php app/nut database:export -d dbdump/ -n`.

### Expected result

A bunch of files in `dbdump/`.

### Actual result

Failure, despair, destruction raining left and right.

- - -

I'll try to make some time to do a bit more digging after this site has seen its release.

**Comment by doenietzomoeilijk** [comment]
@SahAssar chipped in and mentioned it might be about relations. I had the same suspecion, since `relations` was the last field the exporter encountered before bombing. In the same project, after removing relations for the first few contenttypes, the export does indeed run, until it encounters the first contenttype with relations.

**Comment by doenietzomoeilijk** [comment]
This was fixed by #7812 .

Now there's a separate RFC for how to handle select exports/imports, see #7817 .
